### PR TITLE
Update logo for Jetbrains and add Jump Trading

### LINKED ISSUE #11: Update Jetbrains Logo to recent logo

### PR DESCRIPTION
**Pull request by verwoerd**
Resolves #11.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,5 +187,4 @@ For DOMJudge use the `background_64.zip`.
 | Logo | ID | Name | Status |
 |---|---|---|---|
 | <img src="./others/logos/O-1.svg" width="32" height="32"> | O-1 |  OBJECTION! | :heavy_check_mark: |
-| <img src="./others/logos/O-2.svg" width="32" height="32"> | O-2 |  The Wise Owl | :heavy_check_mark: |
-| <img src="./others/logos/O-3.svg" width="32" height="32"> | O-3 |  The Wise Owl | :heavy_check_mark: |
```
Comment (mpsijm) [inline README.md:190]: O-3 should be ETV, looks like the addition in https://github.com/WISVCH/icpc-logos/commit/b4eb0608f1003380d23dd48f4abd932aef38370e was incomplete (it's also missing from https://github.com/WISVCH/icpc-logos/blob/main/others/organizations.json) :stuck_out_tongue:

**Review by mpsijm — APPROVED** [review]
The new logo's are good! The comment is about something unrelated to this PR :slightly_smiling_face:

File: README.md (hunk 1 above)
Comment (verwoerd) [reply README.md:190]: yep, put a fix for that in #10 since its currently not in the `organizations.yaml` for others.

Comment (mpsijm) [reply README.md:190]: Yep, it'll give a merge conflict now though :stuck_out_tongue:

Comment (verwoerd) [reply README.md:190]: my favorite hobby, rebasing branches to fix merge conflicts :)

Comment (mpsijm) [reply README.md]: That worked :smile: Feel free to merge, I'm not sure if Robert would still like to review :slightly_smiling_face:

**Review by robertdijk — APPROVED** [review]
Looks good!

**Pull request merged**